File: add/codesnippet/CSharp/p-system.diagnostics.ent_1.cs
    protected static void MyOnEntry(Object source, EntryWrittenEventArgs e)
    {
        EventLogEntry myEventLogEntry = e.Entry;
        if (myEventLogEntry != null)
        {
            Console.WriteLine("Current message entry is: '"
                              + myEventLogEntry.Message + "'");
        }
        else
        {
            Console.WriteLine("The current entry is null");
        }
    }

File: add/codesnippet/CPP/p-system.diagnostics.ent_1.cpp
void MyOnEntry( Object^ source, EntryWrittenEventArgs^ e )
{
   EventLogEntry^ myEventLogEntry = e->Entry;
   if ( myEventLogEntry )
   {
      Console::WriteLine( "Current message entry is: '{0}'", myEventLogEntry->Message );
   }
   else
   {
      Console::WriteLine( "The current entry is null" );
   }
}